config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+ resources :users
```

Comment:
This project only can edit, create, update and new. So we can also do this `resources :users, only: [:new, :create, :edit, :update]`

Reply:
Ok. But as your first directive says " it is possible to edit a user in app this is a very important feature for this project", we needed to add :index in the route. so we can have a view to edit a specific user in /users.

Reply:
Okay. That time I saw you just use new, create, edit, update. That's why I just share this to you. It's completely fine if you stick with the previous one.

Reply:
Cool. Thank you very much Dipto !